parm/config/gfs/config.resources.GAEAC6

```diff
@@ -46,7 +60,28 @@ case ${step} in
         export tasks_per_node=$(( max_tasks_per_node / threads_per_task ))
     fi
     ;;
-  *)
 
+  "marinebmat")
+    case ${OCNRES} in
+      "025")
+        tasks_per_node=$(( max_tasks_per_node / 4 )) 
+        ;;
+      *)
```

Comment:
remove the `*)`   It is default behaviour.

Reply:
leaving the *) because it's required by shellcheck.